### DISCUSSION TITLE
fetching data with Zustand error

### DISCUSSION THREAD

**Discussion by nikstaaar** [neuefische/questions#156] [Web] [answered]
i have moved my fetched data into a useStore and now I am getting this error and my entire build isn't running. It was working fine before I implemented Zustand. I would appreciate it if someone would look at this problem :) 

https://github.com/nikstaaar/capstone-project/pull/5#issue-1382120681

**Comment by mbosselmann** [comment, marked as answer]
Hi Niklas,
the import of your store in your `bar.js` is wrong. The import should look like this:
`import useStore from '../hooks/useStore.js';`

But your import is: `import {useStore} from 'zustand';` which does not refer to your store, but to the useStore functionality of zustand itself.

I hope my answer is helpful. :)

**Reply by nikstaaar** [reply]
ah such an obvious mistake. thank you
